docs/DeveloperGuide.adoc

```diff
@@ -583,17 +585,98 @@ Alternative 2 also helps prevent user confusion, as the list that is displayed i
 
 // end::generateschedule2[]
 
+// tag::Cris2[]
```

Comment:
You dont have an ending tag - the entire appendix is now inadvertently in your PPP